.github/docs/wiki/6.-Post-Set-Up.md

```diff
@@ -0,0 +1,108 @@
+
+# enabling https traffic
+
+## using cloudflare full (strict)
+
+first of all you need a cloudflare account, to create one go to <https://dash.cloudflare.com/sign-up>, enter your email and password and click on `create account`
+
+![Step 1](https://github.com/osuAkatsuki/bancho.py/blob/master/.github/images/ssl_cf_1.png)
+
+now you have to enter your domain. this has to be your actual domain (e.g. `banchopy.com` or `banchopy.net`) and mustn't include any domain-specific hostnames (e.g. `www.banchopy.com` or similar)
+
+![Step 2](https://github.com/osuAkatsuki/bancho.py/blob/master/.github/images/ssl_cf_2.png)
+
+then you have to choose your plan, for us it should be enough with the `Free plan`, you can also upgrade later if you need it
+
+![Step 3](https://github.com/osuAkatsuki/bancho.py/blob/master/.github/images/ssl_cf_3.png)
+
+now you'll have to copy the nameservers required by Cloudflare to your domain registrar. once you've done this, click on `check nameservers`
+
+![Step 4](https://github.com/osuAkatsuki/bancho.py/blob/master/.github/images/ssl_cf_4.png)
+
+once you have finished the above you'll have to add some dns records (A records) so that the necessary domains are being pointed to the ip where bancho.py is running.
+
+you can generate the records to import in cloudflare using the script inside the `tools` folder
+
+```sh
+cd tools && ./generate_cf_dns_records.sh && cd..
+```
+
+and on the cloudflare dashboard click Import and Export
+
+![Step 5](https://github.com/osuAkatsuki/bancho.py/blob/master/.github/images/ssl_cf_5.png)
+
+If you use free freenom domains like `.ml`, `.ga`, `.ml`, `.cf`, you probably can't import the dns, this is because they are restricted in the Cloudflare API due to significant abuses, in that case you'll have to add the following dns records manually
+
+<table>
+    <tr>
+        <th>
+        <ul>
+            <li>a.yourdomain.com</li>
+            <li>api.yourdomain.com</li>
+            <li>assets.yourdomain.com</li>
+            <li>c1.yourdomain.com</li>
+            <li>c2.yourdomain.com</li>
+            <li>c3.yourdomain.com</li>
+            <li>c4.yourdomain.com</li>
+            <li>c5.yourdomain.com</li>
+            <li>c6.yourdomain.com</li>
+            <li>ce.yourdomain.com</li>
+            <li>cho.yourdomain.com</li>
+            <li>c.yourdomain.com</li>
+            <li>yourdomain.com</li>
+            <li>i.yourdomain.com</li>
+            <li>map.yourdomain.com</li>
+            <li>osu.yourdomain.com</li>
+            <li>s.yourdomain.com</li>
+            <li>web.yourdomain.com</li>
```

Comment:
[this changes](https://github.com/osuAkatsuki/bancho.py/commit/488b7c9ca33fa539720faf0eaa792d8af1b7af9d) are missing

Reply:
Yeah, I know. Was intending to do that in a different PR, but I guess I can do that here.

Reply:
Anything else missing?

Reply:
other than that lgtm